tests/test-rest-nav-menu-locations.php

```diff
@@ -2,8 +2,14 @@
 
 class WP_Test_REST_Nav_Menu_Locations_Controller extends WP_Test_REST_Controller_Testcase {
 
+	/**
+	 * @var
```

Comment:
Missing type.

Reply:
Fixed in https://github.com/WP-API/menus-endpoints/pull/41/commits/565a06fff290cddae77b2167a2d05b457e85e1d6